libraries/Microsoft.Bot.Builder/TurnContext.cs

```diff
@@ -139,11 +139,11 @@ public ITurnContext OnDeleteActivity(DeleteActivityHandler handler)
         /// </summary>
         /// <param name="textReplyToSend">The text of the message to send.</param>
         /// <param name="speak">Optional, text to be spoken by your bot on a speech-enabled
-        /// channel.</param>
+        /// channel. If no text is give, textReplyToSend will be used.</param>
```

Comment:
Nipticky comments: 
grammar. "If no text is provided, " [...]
verb tense: "will be used" -> is used.